dietpi/dietpi-software

```diff
@@ -6343,6 +6357,23 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 			G_EXEC chmod +x $ha_home/homeassistant-update.sh
 
 		fi
+    
+    software_id=181 # PaperMC
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+      
+      # Make sure user agrees to the EULA
+			G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+			if G_WHIP_YESNO "Do you agree to the Minecraft EULA found at:
+\nhttps://account.mojang.com/documents/minecraft_eula"; then
+        Download_Install 'https://papermc.io/ci/job/Paper-1.16/lastSuccessfulBuild/artifact/paperclip.jar' /opt/papermc/paperclip.jar
+        G_EXEC echo 'eula=true' > /opt/papermc/eula.txt
```

Comment:
This does not work, `G_EXEC` cannot be used with redirects. Run this command from console and check `/opt/papermc/eula.txt` content 😉. Possible workaround:
```
G_EXEC eval 'echo "eula=true" > /opt/papermc/eula.txt'
```
I also didn't find a final good solution for this, probably with G_EXEC_REDIRECT or something as there are general concerns of using `eval`, especially in combination with variables.

Reply:
I noticed that, that’s why I switched to G_CONFIG_INJECT.  I’ll use veal.

Reply:
Done already, yes `G_CONFIG_INJECT` would have been an alternative but the file needs to exist, at least empty.